public/app/features/dashboard-scene/scene/layout-rows/RowsLayoutManager.tsx

```diff
@@ -169,7 +170,21 @@ export class RowsLayoutManager extends SceneObjectBase<RowsLayoutManagerState> i
 
     if (layout instanceof TabsLayoutManager) {
       for (const tab of layout.state.tabs) {
-        rows.push(new RowItem({ layout: tab.state.layout.clone(), title: tab.state.title }));
+        if (isClonedKey(tab.state.key!)) {
+          continue;
+        }
+
+        const conditionalRendering = tab.state.conditionalRendering;
+        conditionalRendering?.clearParent();
+
+        const behavior = tab.state.$behaviors?.find((b) => b instanceof TabItemRepeaterBehavior);
+        const $behaviors = !behavior
+          ? undefined
+          : [new RowItemRepeaterBehavior({ variableName: behavior.state.variableName })];
```

Comment:
feel a behavior is wrong here, it should just a be a normal typed object on TabItem / RowItem. but maybe something we can refactor later.

Reply:
🤔  I think we can refactor afterwards. We can also move the functionality out of the `AutoGridItem` to declutter that one as well. WDYT?